cfme/tests/containers/test_tables_sort.py

```diff
@@ -31,20 +31,24 @@
                          ids=[ti.args[1].pretty_id() for ti in TEST_ITEMS])
 def test_tables_sort(test_item, soft_assert, appliance):
 
-    current_view = navigate_to((test_item.obj if test_item.obj is ContainersProvider
+    view = navigate_to((test_item.obj if test_item.obj is ContainersProvider
         else getattr(appliance.collections, test_item.collection_name)), 'All')
-    current_view.toolbar.view_selector.select('List View')
+    view.toolbar.view_selector.select('List View')
 
-    for col, header_text in enumerate(current_view.entities.elements.headers):
+    for col, header_text in enumerate(view.entities.elements.headers):
 
         if not header_text:
             continue
-        current_view.entities.paginator.set_items_per_page(1000)
+        view.entities.paginator.set_items_per_page(1000)
```

Comment:
Question:
Do we need to set the items per page each time to 1000 or can we just set it to 1000 on the initial navigation

Reply:
good call, I'll move this up